packages/xrpl/package.json

```diff
@@ -67,6 +65,13 @@
   },
   "readmeFilename": "README.md",
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=14"
+  },
+  "devDependencies": {
```

Comment:
why are `devDependencies` in two different places? Shouldn't this be up with the other dev deps?

Reply:
@mvadari What? https://github.com/XRPLF/xrpl.js/blob/cfc3c4d8658be59b49b138fed827253a8a0a8697/packages/xrpl/package.json#L70

Reply:
Oh, I misread what was happening. This should happen in a different PR to avoid large diffs, but IMO the dev dependencies should be right under the dependencies

Reply:
Fixed